__tests__/models/League.js

```diff
@@ -193,6 +193,68 @@ describe("addContestantRoundScores", () => {
     });
 });
 
+describe("getNumberOfRounds", () => {
+    it("should not ever return Number.MAX_VALUE", () => {
+        //Arrange
+        const teamList = [{eliminationOrder: 1}, {eliminationOrder:2}, {eliminationOrder: Number.MAX_VALUE}];
```

Comment:
Note: Talked offline about the use of MAX_VALUE. Currently, we use both 0 and MAX_VALUE to indicate that a person is not eliminated. We will discuss offline and make a choice of how to move forward with a single standard. Any refactor will be a future PR.

Reply:
Noted and opened issue #242 to cover this... but definitely out of scope for this change and probably our current work entirely.